pkg/reconciler/domainmapping/reconciler.go

```diff
@@ -79,13 +81,15 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, dm *v1alpha1.DomainMappi
 		return err
 	}
 
-	// Since we currently assume all DomainMapping References are KServices we
-	// don't do any resolution yet, so we'll just immediately mark the condition.
-	dm.Status.MarkReferenceResolved()
+	// Resolve the spec.Ref to a URI following the Addressable contract.
+	target, err := r.resolveRef(ctx, dm)
+	if err != nil {
+		return err
+	}
```

Comment:
We should error out if the target has a Path (e.g. the MT Broker)

Reply:
👍 done